solcjs

```diff
@@ -45,6 +48,26 @@ function abort (msg) {
   process.exit(1);
 }
 
+if (argv['standard-json']) {
+  if (!solc.supportsStandard) {
+    abort('Compiler does not support Standard JSON I/O');
+  }
+
+  var size = fs.fstatSync(process.stdin.fd).size;
```

Comment:
I don't think we should use `process.stdin.fd`: https://github.com/nodejs/node/issues/7439#issuecomment-228760636